airflow/providers/cncf/kubernetes/utils/pod_manager.py

```diff
@@ -75,6 +76,12 @@ class PodPhase:
     terminal_states = {FAILED, SUCCEEDED}
 
 
+class ContainerNames:
+    """Possible container names for airflow."""
+
+    XCOM_CONTAINER = "airflow-xcom-sidecar"
+
+
```

Comment:
I didn't understand the need of this class, this value is already available in `PodDefaults.SIDECAR_CONTAINER_NAME` which we use in the other methods, it's better to use it in case we change its name in the future

Reply:
Good point. I had missed seeing this. I will undo this change and reuse `PodDefaults`